charts/opentelemetry-operator/README.md

```diff
@@ -0,0 +1,319 @@
+```
+
+Note that `--namespace` option here won't affect on where the OTEL Operator and other resources this chart contains are installed.
+It will only affect on where the Helm chart release info is stored, which is `default` namespace by default.
+
+_See [helm install](https://helm.sh/docs/helm/helm_install/) for command documentation._
+
+## Uninstall Chart
+
+The following command uninstalls the chart whose release name is my-opentelemetry-operator.
+
+```console
+$ helm uninstall my-opentelemetry-operator
+```
+
+_See [helm uninstall](https://helm.sh/docs/helm/helm_uninstall/) for command documentation._
+
+This will remove all the Kubernetes components associated with the chart and deletes the release.
+
+The OpenTelemetry Collector CRD created by this chart won't be removed by default and should be manually deleted:
+
+```console
+$ kubectl delete crd opentelemetrycollectors.opentelemetry.io
+```
+
+If the namespace is created by Helm instead of yourself, please skip this step. If you created the namespace and set
+`createNamespace` to `false`, you should remove it manually:
+
+```console
+$ kubectl delete ns opentelemetry-operator-system
+```
+
+## Upgrade Chart
+
+```console
+$ helm upgrade my-opentelemetry-operator open-telemetry/opentelemetry-operator
+```
+
+Please note that by default, the chart will be upgraded to the latest version. If you want to upgrade to a specific version,
+use `--version` flag.
+
+With Helm v3.0, CRDs created by this chart are not updated by default and should be manually updated.
+Consult also the [Helm Documentation on CRDs](https://helm.sh/docs/chart_best_practices/custom_resource_definitions).
+
+_See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documentation._
+
+## Configuration
+
+The following command will show all the configurable options with detailed comments.
+
+```console
+$ helm show values open-telemetry/opentelemetry-operator
+```
+
+## Install OpenTelemetry Collector
+
+_See [OpenTelemetry website](https://opentelemetry.io/docs/collector/) for more details about the Collector_
+
+Once the opentelemetry-operator deployment is ready, you can deploy OpenTelemetry Collector in our Kubernetes
+cluster.
+
+The Collector can be deployed as one of four modes: Deployment, DaemonSet, StatefulSet and Sidecar. The default
+mode is Deployment. We will introduce the benefits and use cases of each mode as well as giving an example for each.
+
+### Deployment Mode
+
+If you want to get more control of the OpenTelemetry collector and create a standalone application, Deployment would
+be your choice. With Deployment, you can relatively easily scale up the collector to monitor more targets, roll back
+to an early version if anything unexpected happens, pause the collector, etc. In general, you can manage your collector
+instance just as an application.
+
+The following example configuration deploys the Collector as Deployment resource. The receiver is jaeger-receiver and
```

Comment:
You have two styles of naming components: `jaeger-receiver` and `logging exporter`. Either use `Jaeger receiver` and `logging exporter`, or `jaeger-receiver` and `logging-exporter`. This last option seems strange to me though.

Reply:
I will unify them using the former one.